docs/annotate.rst

```diff
@@ -63,7 +67,17 @@ Step-by-step
 
    - The Key-Value Pairs allow you to add lab-book-like additional metadata for the Image. These Key-Value Pairs are also specifically searchable. See :doc:`search-omero`.
 
-#. It is also possible to add Comment, Attachment and Rating to selected Images in the right-hand pane in the ``General`` tab.
+#. For adding of File Attachments:
+
+   - Select one or more objects in the left-hand side tree, such as ``Dataset`` or ``Image``.
+   - Expand the ``Attachments`` harmonica in the right-hand pane. 
+   - Click the plus button.
+
+      |image1b|
+
+   - You can attach any type of file using this function. If you select a file from your local hierarchy, the feature will upload that file to the OMERO.server and save it there. The content of ``.pdf``, ``CSV`` and plain text files is also searchable in OMERO.
```

Comment:
> from your local hierarchy

Do you mean from the local filesystem using the `Browse...` button?

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/fa082fdac0f6da738107403d5e2b9037d462a865